services/ui/ws/window_manager_display_root.cc

```diff
@@ -18,6 +19,21 @@
 namespace ui {
 namespace ws {
 
+namespace {
+
+ServerWindow* CreateExternalWindowDisplayRootWindow(
```

Comment:
Move it to WindowServer::CreateExternalWindowDisplayRootWindow ?

Reply:
it was originally there, but I chose to keep it here, so that it is one less file to tweak. I believe when/if this gets upstreamed, we can get an agreement about it placement.